modules/desktop/src/components/package-card/package-card.svelte

```diff
@@ -6,8 +6,9 @@
   import BgImage from "./bg-image.svelte";
```

Comment:
i am getting feeling we might want to create a separate card for local package. to reduce multiple conditionality

Reply:
Fixed.  And yes, it's much better.